### PR TITLE
Hello World app

### PR DESCRIPTION
**Pull request by carjug**
## What was done
- Fresh Django project named `opre_ops`
- Fresh Django app created inside `opre_ops/opre_ops` called `hello_world`
- Route and view `hello_world` declared that returns an HTTPResponse of "Hello world, it's me Margaret."
- database setup

Visit https://opre-ops-test.app.cloud.gov/hello_world/ for some exciting stuff

## Sources
These are all the sources I can remember visiting in the making of this PR  that might be useful to a reviewer

https://docs.djangoproject.com/en/3.2/intro/tutorial01/
https://stackoverflow.com/questions/22841764/best-practice-for-django-project-working-directory-structure
https://docs.djangoproject.com/en/2.2/ref/settings/#allowed-hosts
https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#minimal-manifest
https://docs.cloudfoundry.org/concepts/roles.html#activeroles
https://github.com/18F/usda-fns-ingest/blob/master/usda_fns_ingestor/usda_fns_ingestor/settings/base.py
https://djangostars.com/blog/configuring-django-settings-best-practices/

## Still to do/work that needs to be done but is potentially outside the scope of this ticket
- Figure out how we want to manage dependencies on the backend 
- Figure out how we want to manage settings between different environments

## Issue
#85

### CODE REVIEW

File: opre_ops/opre_ops/settings.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+]
+
+WSGI_APPLICATION = 'opre_ops.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/3.2/ref/settings/#databases
+
+DATABASES = {
+    'default': dj_database_url.config()
```
Comment (carjug) [inline opre_ops/opre_ops/settings.py:78]: Still working on db config stuff for local, will come back to this tomorrow AM.

File: opre_ops/opre_ops/urls.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path
+from opre_ops.apps.hello_world import views
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('hello_world/', views.hello_world, name='hello_world')
+]
```
Comment (carjug) [inline opre_ops/opre_ops/urls.py:23]: There's a bunch of different ways we can do routes in projects with apps with Django, this was just an easy first feature way, not sure how we want to do this in the future though

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+asgiref==3.4.1
+Django==3.2.5
+pytz==2021.1
+sqlparse==0.4.1
```
Comment (carjug) [inline requirements.txt:4]: I mentioned this in the PR comment, but I think we probably want to manage our Python dependencies differently than a requirements.txt file.

File: start-web.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# todo: django docs say don't use this `runserver` command in prod
+python ./opre_ops/manage.py runserver 0.0.0.0:8080
```
Comment (carjug) [inline start-web.sh:4]: Amy had a good resource to use guinicorn instead of runserver here

**Review by amymok — APPROVED** [review]
LGTM!

File: opre_ops/opre_ops/apps/hello_world/views.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+
+
+def hello_world(request):
+    return HttpResponse("Hello world, it's me Margaret!")
```
Comment (alexsoble) [inline opre_ops/opre_ops/apps/hello_world/views.py:6]: 😄 😄

File: manifest.yml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
+applications:
+    - name: opre-ops-test
+      buildpacks:
+        - https://github.com/cloudfoundry/python-buildpack.git#v1.7.44
```
Comment (alexsoble) [inline manifest.yml:5]: If we pin the precise version here, will we still get the benefit of cloud.gov sending us an email telling us to re-stage the application when a newer version of the buildpack is released? 

Looking at: https://cloud.gov/docs/getting-started/concepts/#buildpacks.

Comment (carjug) [reply manifest.yml:5]: Ah yeah, we can just bring in the `python_buildpack` if you think that's best, @alexsoble. I didn't know this was an option!

**Review by alexsoble — APPROVED** [review]
LGTM! 🎉 `:hello-world:`

**Pull request merged**